lib/iris/coords.py

```diff
@@ -1021,6 +1021,13 @@ def _guess_bounds(self, bound_position=0.5):
 
             This method only works for coordinates with ``coord.ndim == 1``.
 
+        .. note::
+
+            If `iris.FUTURE.clip_latitudes` is True, then this
+            method will clip the coordinate bounds to the range
+            [-90, 90] for latitude and grid_latitude coordinates
+            measured in degrees.
```

Comment:
@rhattersley Is it also worth explicitly saying that "... it clips iff *all* points are within [-90, 90]"